string topic
---
string previous_topic
bool success
bool new_topic